yb-voyager/cmd/getDataMigrationReportCommand.go

```diff
@@ -150,6 +155,9 @@ func getDataMigrationReportCmdFn(msr *metadb.MigrationStatusRecord) {
 	if source.DBType == POSTGRESQL {
 		exportSnapshotStatus, err = exportSnapshotStatusFile.Read()
 		if err != nil {
+			if errors.Is(err, fs.ErrNotExist) {
+				return
```

Comment:
Shall we exit with a msg here along the lines of "export has not started yet. Try running after export has started." (same for export-data-status as well)